Update tech lead persona - Sebastian (#489)
kyma-project/community: manifesto/personas.md

| Accountability/Decisiveness (1-6) | 💪 💪 💪 💪 💪 💪                                                                                                                                                                                                                                                                                                                                                                                                                                         |
| Slogans                           | _I know what I want... not yet sure how to achieve it._                                                                                                                                                                                                                                                                                                                                                                                                   |

### GAINS

- :green_heart: I need to evaluate Kyma fast (within 1 day).
- :green_heart: I love tutorials, sample code, recepies that refer to my use cases (i.e authentication/authorisation flows).

> - :green_heart: I want to tick off on product (industry) standards for free (i.e security, accessibility, observability, tracing, etc).

Reviewer: ```suggestion
- :green_heart: I want to ensure product (industry) standards for free (i.e security, accessibility, observability, tracing, etc).
```
Is that what you mean? This term is not very clear and might not be comprehensible to most readers.

> - :green_heart: I want future proof platform - based on references.

Reviewer: ```suggestion
- :green_heart: I want a futureproof platform, based on references.
```
Again, is that what you had in mind?

> - :green_heart: With Kyma I want to connect to systems I know (public and internal systems).

Reviewer: ```suggestion
- :green_heart: With Kyma, I want to connect to systems I know (public and internal ones).
```

- :green_heart: I need a trial version of Kyma before I can recommend it.

> - :green_heart: I need agility in architecture designs - I want to replace individual modules if needed.

Reviewer: ```suggestion
- :green_heart: I need agility in architecture design. I want to replace individual modules if needed.
```
I would treat it as an abstract noun here, unless you're talking about a particular design, not the process.

- :green_heart: I want a painless upgrade path.
- :green_heart: I want operational support.

### PAINS

> - :broken_heart: I'm giving up on a project if success stories, references are missing

Reviewer: ```suggestion
- :broken_heart: I give up on a project if success stories and/or references are missing.
```
`I'm giving up` would mean that he's doing it now. I believe what you mean here is a general rule for him.

> - :broken_heart: I'm giving up on a project if documentation (inl. tutorials) is missing.

Reviewer: ```suggestion
- :broken_heart: I give up on a project if documentation (incl. tutorials) is missing.
```

> - :broken_heart: I don't want to take extra care about security/monitoring/tracing...

Reviewer: ```suggestion
- :broken_heart: I don't want to take extra care of security, monitoring, tracing, etc.
```
